.github/workflows/ci-testing.yaml

```diff
@@ -54,3 +54,7 @@ jobs:
       - name: Run Python functional tests
         shell: bash
         run: PYTHONPATH=$(pwd)/src python -m pytest tests/functional_tests/ --color=yes -n auto
+
+      - name: Run Python unit tests
+        shell: bash
+        run: PYTHONPATH=$(pwd)/src python -m pytest tests/unit_tests/ --color=yes -n auto
```

Comment:
Should we run the unit tests before the functional ones?

Reply:
A very good point. Yes, we should.